dallinger/recruiters.py

```diff
@@ -561,6 +555,20 @@ def open_recruitment(self, n=1):
             "message": "HIT now published to Amazon Mechanical Turk",
         }
 
+    def assign_experiment_qualifications(self, worker_id, qualifications):
+        """Assigns MTurk Qualifications to a worker. Do nothing if qualification
+        assignment is globally disabled.
+        """
+        if not self.qualification_active:
```

Comment:
I might be inclined to move this check to the experiment also. Let the recruiter be ignorant about all the policies about if and when to assign qualifications.

Reply:
Agreed. The recruiter is just a middle man, the experiment should make the decisions.